db/submission_stats.py

```diff
@@ -39,6 +39,8 @@
     LOWLEVEL_TOTAL_UNIQUE: "Total (unique)",
 }
 
+NORMALIZATION_SAMPLE_SIZE = 10000
```

Comment:
I think this should be in the other file?

Reply:
Good catch! Thank you.

Reply:
Turns out it just needs to be removed from this one